File: Negocio/ParametroCustom.vb
Namespace LiquidacionSueldos
    Namespace Negocio
        Partial Public Class Parametro
            Public Function ObtenerValor(ByVal Nombre As String) As String
                ocdGestor = New Datos.Gestor
                Tabla = New DataTable

                Try
                    Tabla = ocdGestor.EjecutarReader("[Parametro.ObtenerTodoBuscarxNombre]", New Object(,) {{"@Nombre", Nombre}})
                Catch ex As Exception
                    Throw ex
                End Try

                If Tabla.Rows.Count = 0 Then
                    Return ""
                Else
                    Return Tabla.Rows(0)("Valor").ToString()
                End If
            End Function
        End Class
    End Namespace
End Namespace

File: NegocioCSharp/ParametroCustom.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
namespace LiquidacionSueldos
{
    namespace Negocio
    {
        public partial class Parametro
        {
            public string ObtenerValor(string Nombre)
            {
                ocdGestor = new Datos.Gestor();
                Tabla = new DataTable();

                try
                {
                    Tabla = ocdGestor.EjecutarReader("[Parametro.ObtenerTodoBuscarxNombre]", new object[,] { {
                        "@Nombre",
                        Nombre
                    } });
                }
                catch (Exception ex)
                {
                    throw ex;
                }

                if (Tabla.Rows.Count == 0)
                {
                    return "";
                }
                else
                {
                    return Tabla.Rows[0]["Valor"].ToString();
                }
            }
        }
    }
}







